### ISSUE TITLE
Add jQuery to package.json dependencies

### ISSUE THREAD

**Issue by lawlesscreation** [bgrins/spectrum#539]
Hi,

If I understand `package.json` dependencies correctly, as jQuery is required for Spectrum then it should be listed as a dependency?

I'm happy to raise a PR for this. Is there any particular version of jQuery required?

**Comment by ephemere000** [comment]
I agree with **lawlesscreation**. I installed the package in my native JS application only to find out it's jQuery dependency.  :(

I had read the Devtools section of http://bgrins.github.io/spectrum/ and it said specifically:  Then I pulled the jQuery dependency out of a branch and I submitted a patch to the WebKit project. Moreover package.json does not contain the dependency, JQuery is therefore not on the GitHub dependencies section.

I assumed wrongly that JQuery was removed since it's integration with the WebKit. I guess it was just a separate specific branch.

Finally, I saw on the http://bgrins.github.io/spectrum/ page the header: "The No Hassle JQuery Colorpicker". We obviously don't often look at the stylized small header of a web page, we mainly focus on the big headers and main content. I discovered it with the search in page tool!

It looks like a great and useful tool. If ever that vanilla JS branch were available, I would give it a shot! Most programmers do not use jQuery on any new project since HTML 5.
:)
